Contact form & finish mobile version pair programming

JuanDiegoGuerra opened the pull request:
>Pair Programming session:
- Added Contact-form section in the index.html file.
- Updated mobile_first.css styles for the new section and its elements.
- Fixed submit-button position from contact-section.
- Fixed Get-in-touch button position and its properties.
- Fixed bottom-bar line position and its properties.

![pair_programming_request](https://user-images.githubusercontent.com/45726631/228384483-50f8364c-af10-4f44-b48e-e5f7c492198a.JPG)

>Coding partners **[@Alejuda @enmanuelbayen]**

JuanDiegoGuerra committed:
Update index.html

Fixed position of some elements.
Update the action form for the contact-me section.

JuanDiegoGuerra committed:
Update mobile_first.css

Fixed styles for the updated elements in the index.html file

JuanDiegoGuerra committed:
Update .gitignore

Added the file that I use for my workspace

Alejuda committed:
Update mobile-first.css

Added inter font-family. Fixed linter errors.

enmanuelbayen committed:
Mobile_first.css fix

enmanuelbayen committed:
Update with styles

changed font-style for input, fixed button position and minor fix

JuanDiegoGuerra committed:
Update README.md

Added my coding partners' github profile links

JuanDiegoGuerra committed:
Update mobile_first.css

Fixed linter errors.

ouasamine commented on index.html:
```diff
@@ -160,20 +160,21 @@ <h3 id="h3-lang">
-           <textarea name="message" rows="3" maxlength="500" required placeholder="Hey I’d like to connect to..."></textarea>
-         </form>
+           <textarea name="message" rows="4" maxlength="500" required placeholder="Hey I’d like to connect to..."></textarea>
+           <button type="submit" name="submit" id="contact-button">Get in touch</button>
```
- [ ] Nice work on the button, it looks exactly like the Figma except for a tiny detail that was missed which is the button interactions 🔽 please implement these interactions they are essential and improve accessibility

![2023-03-28_230859](https://user-images.githubusercontent.com/104319462/228387123-cbfb79be-318f-41e4-9f0b-cff2fb3b454f.png)

- [ ] [OPTIONAL] Currently when I hover over the button my cursor icon doesn't switch to the hand icon, please put a `cursor: pointer` property to your button, its a common practice in interfaces developments, and users will always expect this behaviour when they hover over buttons.

ouasamine commented on index.html:
```diff
@@ -160,20 +160,21 @@ <h3 id="h3-lang">
+            <p>jdxelric@gmail.com</p>
         </div> 
-        <ul>
+        <ul class="footer-icons">
```
- [ ] Please remove this bar 🔽 , it's actually a phone navigation bar and is not part of the design.

![2023-03-28_231428](https://user-images.githubusercontent.com/104319462/228387761-794398ac-2a23-49d6-824a-967c3f42316b.png)

ouasamine reviewed (changes requested):
Hi @JuanDiegoGuerra

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!

![](https://media.giphy.com/media/t8YPf3ACKeouLzEIvr/giphy.gif)

## Highlights
- Nice Ui, it looks exactly like the Figma ✔️ 👏🏽 
- Your food is well organised good job 👏🏽 👍🏽 

## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

JuanDiegoGuerra committed:
Updated Required Changes

-Removed the bar from the bottom of the page, it was actually a phone navigation bar and wasn't part of the design.
-Added the [OPTIONAL] cursor pointer for the "Get in touch" button, so can change the color when hover it.

JuanDiegoGuerra commented:
- On the last commit, I made the required changes.

![required_changes](https://user-images.githubusercontent.com/45726631/228410367-07109084-4514-4a0a-9246-edef51ec2b8d.gif)

topeogunleye commented on index.html:
```diff
@@ -160,27 +160,27 @@ <h3 id="h3-lang">
-           <textarea name="message" rows="3" maxlength="500" required placeholder="Hey I’d like to connect to..."></textarea>
-         </form>
+           <textarea name="message" rows="4" maxlength="500" required placeholder="Hey I’d like to connect to..."></textarea>
+           <button type="submit" name="submit" id="contact-button">Get in touch</button>
```
- [ ] As suggested by the previous reviewer I kindly suggest that you add the correct different button interactions including hover and active (when it's pressed) as shown in the Figma template below:

![image](https://user-images.githubusercontent.com/52934128/219883731-3d32d6f5-53ac-4128-a0c8-1a7595de03e5.png)

the button on the pressed state has the `background: #008552;` and on hover has the `background: #36B37F;` not `background: #033700;` as you currently have it.

### EXAMPLE:
**for pressed state**
```
button:active {
   background-color: #008552;
}
```
**for hover state**
```
button:hover {
    background-color: #36B37F;
}
```

topeogunleye commented on css/mobile_first.css:
```diff
@@ -512,41 +515,35 @@ h3 {
-  width: 153px;
-  padding-top: 12px;
-  padding-bottom: 12px;
+#contact-button {
```
- [ ] As suggested by the previous reviewer I kindly suggest that you add the correct different button interactions including hover and active (when it's pressed) as shown in the Figma template below:

![image](https://user-images.githubusercontent.com/52934128/219883731-3d32d6f5-53ac-4128-a0c8-1a7595de03e5.png)

the button on the pressed state has the `background: #008552;` and on hover has the `background: #36B37F;` not `background: #033700;` as you currently have it.

### EXAMPLE:
**for pressed state**
```
button:active {
   background-color: #008552;
}
```
**for hover state**
```
button:hover {
    background-color: #36B37F;
}
```

topeogunleye commented on css/mobile_first.css:
```diff
@@ -241,10 +245,8 @@ h3 {
 #about-me-section-4 {
-  margin-bottom: 30px;
   border: none;
   width: 314px;
```
- [ ] [OPTIONAL] I kindly suggest that you make use of percentages for widths instead of px as this is ensure your screen with adjusts effectively for other device. Kindly see a video below for more explanation.

https://www.loom.com/share/d11115e0d85d48f6ba73427940dfd5b7

topeogunleye reviewed (changes requested):
Hi @JuanDiegoGuerra, 

Great work on making the changes requested by a previous reviewer 👏 
You've done well implementing some of the requested changes, but there are still some which aren't addressed yet.


 ## Highlights
* Good commit messages. :heavy_check_mark:
* No linter errors. :heavy_check_mark:
* Followed Github flow :heavy_check_mark:
* Professional README file. :heavy_check_mark:
* PR has a good title and summary :heavy_check_mark:

 ## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

You can also consider:
- Please kindly check my inline comments.

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag @topeogunleye in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**


------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

JuanDiegoGuerra committed:
Update mobile_first.css

Updated buttons interactions including hover and active (when it's pressed).

JuanDiegoGuerra committed:
Update mobile_first.css

Fixed linters error...

JuanDiegoGuerra commented:
- Made the required changes from last review.
 ºThe button interactions are working fine on the 3 states:

![fixed_button_interactions](https://user-images.githubusercontent.com/45726631/228442555-efe26bf7-ae49-4537-88c4-00e416230fc5.gif)

topeogunleye commented:
> * Made the required changes from last review.
>   ºThe button interactions are working fine on the 3 states:
> 
> ![fixed_button_interactions](https://user-images.githubusercontent.com/45726631/228442555-efe26bf7-ae49-4537-88c4-00e416230fc5.gif) [ ![fixed_button_interactions](https://user-images.githubusercontent.com/45726631/228442555-efe26bf7-ae49-4537-88c4-00e416230fc5.gif) ](https://user-images.githubusercontent.com/45726631/228442555-efe26bf7-ae49-4537-88c4-00e416230fc5.gif) [ ](https://user-images.githubusercontent.com/45726631/228442555-efe26bf7-ae49-4537-88c4-00e416230fc5.gif)

@JuanDiegoGuerra, Great Job Guys. Keep it up.

shella12 reviewed (approved):
Hi @JuanDiegoGuerra, 


## Status : APPROVED 🥳 🥳 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 

<img src="https://media0.giphy.com/media/3kuSo744UIPJjcJUEn/giphy.gif"/>

## Highlights

- Linters are passing ✔️ 
- README is professional 👍 
- Great design implementation 👏 

## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._
